src/Blocks/CarouselHeader.php

```diff
@@ -123,6 +123,11 @@ public static function enqueue_frontend_assets(): void
                 '2.0.8',
                 true
             );
+            wp_script_add_data(
+                'hammer',
```

Comment:
Since Hammer is being used twice (Carousel Header and Gallery), the idea is to install it through `NPM` like we do in [this PR](https://github.com/greenpeace/planet4-master-theme/pull/2618/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R56).